scala/lambdas/ingest-parsed-court-document-event-handler/src/test/scala/uk/gov/nationalarchives/ingestparsedcourtdocumenteventhandler/FileProcessorTest.scala

```diff
@@ -299,8 +299,9 @@ class FileProcessorTest extends AnyFlatSpec with TableDrivenPropertyChecks {
               val assetId = treMetadata.parameters.TDR.`UUID`
               val fileName = treFileName.split('.').dropRight(1).mkString(".")
               val folderTitle = if titleExpected then Option(expectedFolderTitle) else None
+              val expectedSeries = series <+> Option("Unknown")
```

Comment:
Should `series` be `potentialSeries`?

Reply:
I think it was already like that but I've changed it to potentialSeries.